compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -21,7 +21,17 @@ namespace mlir::iree_compiler::AMDAIE {
 
 void buildAMDAIETransformPassPipeline(OpPassManager &pm) {
   addCommonTargetExecutablePreprocessingPasses(pm);
+  // TODO: Current we don't include C++ equivalent of Transform
+  //       dialect scripts. We are thus guarding their inclusion
+  //       with a bool `useCPlusPlusTransformPasses` which would
+  //       be used during development efforts. Once the C++ passes
+  //       are ready, we will include these passes by default and
+  //       take away the guarding.
+  bool useCPlusPlusTransformPasses = false;
+  if (useCPlusPlusTransformPasses)
```

Comment:
IIUC you want to set this to true when using the matmul_fill_spec_pad_new.mlir script? If thats the case I think what you want is a global option that you can set from the command as true, here is an example, 
https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Dialect/HAL/Target/LLVMCPU/LLVMCPUTarget.cpp#L56-L59
so then when running with pad_new.mlir you can set it to true